modules/vaos/spec/services/configuration_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe VAOS::Configuration do
+  describe '#service_name' do
+    it 'has a service name' do
+      expect(VAOS::Configuration.instance.service_name).to eq('VAOS')
+    end
+  end
+
+  describe '#connection' do
+    it 'returns a connection' do
+      expect(VAOS::Configuration.instance.connection).not_to be_nil
+    end
+  end
+
+  describe '#mock_enabled?' do
+    context 'when Settings.va_mobile.mock is true' do
```

Comment:
The `before` blocks here are each used by a single example. Seems like inlining them into the example would make each example clear and independent.

Reply:
It would be more concise but I believe the context block is for the state the spec is running in. In this case flipping the mock setting. Then the it block is for confirming the expectation. Ideally in one line but obviously that's not always possible 😄